showcase/styles.css

```
@@ -0,0 +1,203 @@
+  border: 1px solid var(--border);
+}
+.cta-banner h2 { font-size: clamp(1.5rem, 3vw, 2.25rem); margin-bottom: 12px; }
+.cta-banner p { margin-bottom: 24px; }
+
+/* ---------- Footer ---------- */
+footer { border-top: 1px solid var(--border); padding: 40px 0; }
+footer .container { display: flex; align-items: center; justify-content: space-between; gap: 16px; flex-wrap: wrap; }
+footer p { font-size: 0.8125rem; }
+
+/* ---------- Reveal ---------- */
+@keyframes fadeInUp { from { opacity: 0; transform: translateY(20px); } to { opacity: 1; transform: translateY(0); } }
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)